src/javascript/JContent/EditFrame/Create.jsx

```diff
@@ -42,8 +42,11 @@ function reposition(element, currentOffset, setCurrentOffset) {
     }
 }
 
-const useElemAttributes = ({element, parent, isInsertionPoint}) => {
-    const nodePath = (element.getAttribute('path') === '*' || isInsertionPoint) ? null : element.getAttribute('path');
+const useElemAttributes = ({element, parent}) => {
+    const isInsertionPoint = element.getAttribute('type') !== 'placeholder';
```

Comment:
Need to check here if insertionPoint is an original create button or not, otherwise it breaks create button for specific child nodes.